.changeset/giant-suns-lick.md

```diff
@@ -0,0 +1,5 @@
+---
+
+---
+
+Update v4 release script to release @4 not @latest
```

Comment:
No need for a changeset in this PR as this is not a change that affects the SDKs themselves, just the release process